deb/deb.go

```diff
@@ -488,8 +439,21 @@ func copyToTarAndDigest(file *files.Content, tw *tar.Writer, md5w io.Writer) (in
 	return file.Size(), nil
 }
 
-func createChangelogInsideDataTar(tarw *tar.Writer, md5w io.Writer,
-	created map[string]bool, info *nfpm.Info,
+func withChangelogIfRequested(info *nfpm.Info) *nfpm.Info {
+	if info.Changelog != "" {
```

Comment:
nitpick but could change the if to == "" and return early to lower the ident scope of the rest of the code

Reply:
Wow, I'm normally super picky about always returning early, I can't belief I missed this one.